locations/spiders/aramark_uniform_services_us.py

```diff
@@ -1,12 +1,13 @@
 from scrapy import Spider
 from scrapy.http import JsonRequest
 
+from locations.categories import Categories, apply_category, apply_yes_no
 from locations.dict_parser import DictParser
 
 
 class AramarkUniformServicesUSSpider(Spider):
     name = "aramark_uniform_services_us"
```

Comment:
If Aramark is rebranding themselves, we should also update the spider name, class name, and file name too.

Reply:
I think you're right, and the source data differentiating names Aramark vs. Verstis is just a case of "haven't had a chance to rebrand all locations yet". But in the future, https://www.aramark.com/our-services/uniform-services makes it sound like Aramark won't deliver any uniform or cleanroom services under an Aramark brand, that will be done solely by Verstis instead. Aramark then keeps food services, facilities management, etc services under their existing brand.

Reply:
Yea, it does look like there will still be Aramark places. Vestis is a whole new company that spun off, so at some point we'll probably make a scraper for them and revisit this one.

(Vestis Pittsburgh should probably file a trademark infringement lawsuit... https://www.vestispgh.com/)

Reply:
I changed the filename, classname, spidername, and urls as they vestis.com/graphql now works. Thus I think this should be good to merge.